open_spiel/CMakeLists.txt

```diff
@@ -197,6 +202,7 @@ add_subdirectory (examples)
 add_subdirectory (games)
 add_subdirectory (game_transforms)
 add_subdirectory (tests)
+add_subdirectory (contrib)
```

Comment:
@lanctot with all the new optional deps it starts to feel like they are "all over the place" :) I would either put tensorflow into tensorflow/ directory, or if we make contrib directory we should put every third party code here. This should probably include also: libtorch, eigen, absl. Not sure what about bots, for games I think it is fine where they are. WDYT?

Reply:
I think absl is separate from libtorch and eigen in this example since we download it.

Nothing is third_party in the libtorch and eigen cases, that's all our code. They are associated with optional dependencies, yes, but I don't see a better place to put them? 

I also don't see how renaming contrib to tensorflow is much help but we can do that.

I personally don't see much benefit to this right now.

Reply:
Ok. Can you @mrdaliri  please change contrib -> tensorflow?

Reply:
@michalsustr Done.